bigtable/tests/unit/test_batcher.py

```diff
@@ -94,6 +94,22 @@ def test_mutate_rows(self):
 
         self.assertEqual(table.mutation_calls, 1)
 
+    def test_mutate_rows_timeout(self):
+        table = _Table(self.TABLE_NAME)
+        mutation_batcher = MutationsBatcher(table=table, timeout=600000)
+
+        row = DirectRow(row_key=b"row_key")
+        row.set_cell("cf1", b"c1", 1)
+        row.set_cell("cf1", b"c2", 2)
+        row.set_cell("cf1", b"c3", 3)
+        row.set_cell("cf1", b"c4", 4)
+
+        mutation_batcher.mutate(row)
+
+        mutation_batcher.flush()
+
+        self.assertEqual(table.mutation_calls, 1)
+
```

Comment:
This test won't be needed, but we do need a test for `Table.mutate_rows` which shows that the configure timeout is passed through to the worker.